docs/Project-Configuration/Project-Configuration-Files.md

```diff
@@ -435,33 +435,51 @@ Fields:
 -   `auto_update`: if true, the latest revision for the module will be
     dynamically retrieved for each Github PR and CLI patch submission
 
-### Pre, Post, and Timeout
+### Pre and Post
 
-All projects can have a `pre` and `post` field which define a list of
-command to run at the start and end of every task that isn't in a task
-group. For task groups, `setup_task` and `teardown_task` will run instead of
-`pre` and `post`. These are incredibly useful as a place for results commands or
-for cleanup and setup tasks.
+All projects can have a `pre` and `post` field which define a list of commands
+to run at the start and end of every task that isn't in a task group. For task
+groups, `setup_task` and `teardown_task` will run instead of `pre` and `post`
```

Comment:
I think it would be helpful to add a few words about "teardown_group" in case someone is grepping to figure out the difference between post, teardown_task and teardown_group.